### ISSUE TITLE
Error parsing enums during generate

### ISSUE THREAD

**Issue by B3none** [steebchen/prisma-client-go#1350]
I'm running: `go run github.com/steebchen/prisma-client-go generate`

This enum:
```prisma
enum ServerOS {
  Unknown // corresponds to ServerOS_Unknown
  Linux // corresponds to ServerOS_Linux
  Windows // corresponds to ServerOS_Windows
  Mac // corresponds to ServerOS_Mac
}
```

Seems to be getting changed in the `db_gen.go` to `ServerOs` and it's causing me the following errors:
```
# my-lovely-project/db
db/db_gen.go:753:23: undefined: ServerOs
db/db_gen.go:4894:52: undefined: ServerOs
db/db_gen.go:4914:55: undefined: ServerOs
db/db_gen.go:4929:65: undefined: ServerOs
db/db_gen.go:4945:56: undefined: ServerOs
db/db_gen.go:4954:53: undefined: ServerOs
db/db_gen.go:4968:62: undefined: ServerOs
db/db_gen.go:4975:56: undefined: ServerOs
db/db_gen.go:4989:65: undefined: ServerOs
db/db_gen.go:4996:52: undefined: ServerOs
db/db_gen.go:4996:52: too many errors
```

At the minute I have a workaround, but I have to put this in a build step:
```Dockerfile
# Ensure db/db_gen.go is present and then apply the sed command to replace all occurrences
RUN if [ -f db/db_gen.go ]; then sed -i 's/ServerOs/ServerOS/g' db/db_gen.go; else echo "db/db_gen.go not found"; exit 1; fi
```

**Comment by steebchen** [comment]
Thanks, fixed via https://github.com/steebchen/prisma-client-go/pull/1359 and release in v0.41.0

**Comment by B3none** [comment]
> Thanks, fixed via #1359 and release in v0.41.0

Awesome thank you! 🎉
